### DISCUSSION TITLE
FadeOut

### DISCUSSION THREAD

**Discussion by mgretz** [matteobruni/tsparticles#2598] [General]
Ist it possible to fade the Particles on stop? So actually i let particles spawn after a mouse click and stop it after 2 seconds. But the stop is immediatley and feels too abrupt.

A FadeOut for the Stop event would be nice.

**Comment by matteobruni** [comment]
Hi @mgretz,

You can achieve something like a fade out effect using opacity animation like [this example](https://codepen.io/matteobruni/pen/ZEJgJoq)

If you change the `particles.opacity.animation.speed` the particles will fade out faster or slower.
